### ISSUE TITLE
Decide on list of "applied_to" options

### ISSUE THREAD

**Issue by spectranaut** [w3c/aria-at#108]
We have been using both lists of specific screen reader and also categories.

- Specific ATs: 'JAWS', 'NVDA', 'VoiceOver', and 'Orca'
- Categories: 'Screen readers', 'Desktop screen readers'

Do we want to have a mix of categories and specific lists? If so, should we decide some norms for the relationships between the categories and specific screen readers?

**Comment by mcking65** [comment]
Yes, there are specific relationships; it is a hierarchy of assistive technologies. This will become more important as we add more assistive technologies.

Here are the values and hierarchy I propose:

* Screen Readers
   * Desktop Screen Readers
      * JAWS
      * NVDA
      * VoiceOver for MacOS
      * Narrator
      * Orca
      * ChromeVOX
   * Mobile Screen Readers
      * VoiceOver for iOS
      * TalkBack
      * VoiceAssistant

I prefer that we write "VoiceOver for MacOS" rather than MacOS VoiceOver" because it is usually more convenient to hear the screen reader name first.

When validating and using the applies to field, if the field contains "Screen Readers", then the assertions apply to all screen readers. If it contains "Desktop Screen Readers", then they apply to all desktop screen readers. We need to be able to provide comma separated value lists as well, such as "JAWS, NVDA"".

Values like "JAWS, NVDA, Mobile Screen Readers" could also be possible but are not likely.

**Comment by spectranaut** [comment]
To do for Val: integrate this information into the test design.
